ui/src/sources/components/SourceStep.tsx

```diff
@@ -35,6 +34,8 @@ import {SUPERADMIN_ROLE} from 'src/auth/Authorized'
 // Types
 import {Source, Me} from 'src/types'
 
+const isNewSource = (source: Partial<Source>) => !source.id
```

Comment:
what do you think about moving this to a utils file?

Reply:
I think this is so small that it is not necessary. I essentially made this a function in order to avoid commenting it, and it doesn't make sense out of the context of this page.